src/Elastic.Apm/Logging/ScopedLogger.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Elastic.Apm.Logging
 {
-	internal class ScopedLogger : IApmLogger
+	public class ScopedLogger : IApmLogger
```

Comment:
Any specific reason making it public? My only concern with making this public is that this class isn't documented. I think public things should have proper XML docs.

Reply:
You are right - I've changed it back to `internal`.